src/styles/blocks/image.css

```diff
@@ -0,0 +1,12 @@
+.image {
+  display: inline-block;
+}
+
+.photo {
+  width: 160px;
+  height: 134px;
+}
+
```

Comment:
use proper naming convension

Reply:
All fullCard styles moved to fullCard.css list. I understand my mistake.
Used proper name for image photo class inside image block